components/tag-mgt/org.wso2.carbon.identity.tag.manager.api.rest/org.wso2.carbon.identity.tag.manager.api.rest.v1/.openapi-generator/tag_manager.yaml.sha256

```diff
@@ -0,0 +1 @@
+3f7fcdb9c5922a03603c9cd6a807d2b5948bdf51b944365c479282013ef433d4
```

Comment:
Add .openapi* to .gitignore file and remove .openapi-generator/ files from the PR

Reply:
removed in 52d3387